Scripts/VerifyNoBuildSettings.swift

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env xcrun --sdk macosx swift
+
+// This script is meant to be called from an Xcode run script build phase
+// It verifies there are no buildSettings embedded in the Xcode project
+// as it is preferable to have build settings specified in .xcconfig files
+
+import Darwin
+import Foundation
+
+func reportError(_ message: String, file: String = "", line: Int = 0) {
+  let formatted = "\(file):\(line): error: \(message)\n"
```

Comment:
![untitled](https://user-images.githubusercontent.com/1836384/43514939-fb75d668-9589-11e8-965a-e87565800641.png)
Special format for XCode integration. Now errors are countable and clickable.

Reply:
SO NICE!